dso-l2/src/main/java/com/tc/objectserver/handler/ReplicationSenderMessage.java

```diff
@@ -0,0 +1,15 @@
+package com.tc.objectserver.handler;
```

Comment:
I wonder if we can get rid of this new class and just use SyncActivityMessage as the type then handle flushes another way.